R/ee_roll_stat.R

```diff
@@ -0,0 +1,61 @@
+
+
+
+
```

Comment:
Why all these blank lines? Maybe you can run styler on these to make them a bit more readable

Reply:
good idea!

Reply:
ran styler on upstream branch

Reply:
Ran styler on `_targets.R` , `exploration/` and `R/`  in this branch 290eeb67e125c5464477bd27a8f6b19ea02eda13